style.css

```
@@ -0,0 +1,41 @@
+body {
+    padding-left: 100px;
+}
+
+.little-image {
+    height: 40px;
+    margin: 15px;
+    margin-left: -5px;
+    width: 40px;
+}
+
+.little-image:active {
+    margin-bottom: 1%;
+}
+
+.meme-image {
+    height: 350px;
+    visibility: visible;
+    width: 350px;
+}
+
+.meme-image-container {
+    background-color: rgb(255, 255, 255);
+    border: 1px solid rgb(0,0,0);
+    height: 350px;
+    margin-top: 15px;
+    position: relative;
+    width: 350px;
+}
+
+.meme-text {
+    color: white;
+    font-size: 30px;
+    left: 7%;
+    position: absolute;
+    text-align: center;
+    text-shadow: -1px 1px 2px black, 1px -1px 2px black;
+    top: 55%;
+    width: 80%;
+    word-break: break-all;
```

Se utilizar `normal`, ele vai quebrar por palavras, e não por letras!
```suggestion
    word-break: normal;
```